### DISCUSSION TITLE
Add an external EKS cluster to ArgoCD

### DISCUSSION THREAD

**Discussion by MonkzCode** [argoproj/argo-cd#13234] [Q&A]
Hello!

Please, help me with argocd configuraton.
I have Argocd installed on my local server in K8s, now i need to connect to it an external EKS cluster. 
I'm trying to add secret:

`kubectl create secret generic argocd-eks-cluster --from-literal=name=EKS_CLUSTER --from-literal=server=https://134214521451234214532452345.yl4.eu-north-1.eks.amazonaws.com --from-literal=config='{"awsAuthConfig": {"clusterName": "arn:aws:eks:eu-north-1:111111111:cluster/eks_cluster", "roleARN": "arn:aws:iam::111111111:role/AWSEKSRole"}}' -n argocd && kubectl label secret argocd-eks-cluster argocd.argoproj.io/secret-type=cluster`

I see new cluster, but it doesn't connected to eks.

Maybe i need to pass some IAM credentials into awsAuthConfig? Or i miss something else?
Sad, that declarative setup documentation have only GKE sample :/

please, help

**Comment by MonkzCode** [comment]
Anyone? I hope there is good support community here

**Reply by jannfis** [reply]
If in doubt, you can add the cluster using `argocd cluster add` and then have a look at the secret that was created.

**Comment by MonkzCode** [comment]
I figured it out, i'm closing this issue

**Reply by mukundjalan** [reply]
@MonkzCode could you please explain what did you find?
